docs/components/modeler/using-web-and-desktop-modeler-together.md

```diff
@@ -0,0 +1,80 @@
+| Can load multiple templates defined in a single file. | Only supports one template per file. |
+
+To maintain compatibility, avoid defining multiple templates in a single file.
+
+### Shared element templates
+
+| Desktop Modeler                                                                                                                                    | Web Modeler                                                                                                                                                         |
+| -------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Templates can be installed as [global templates](/components/modeler/desktop-modeler/element-templates/configuring-templates.md#global-templates). | Templates are [published to the organization](/components/connectors/manage-connector-templates.md#manage-published-connector-templates) for reuse across projects. |
+
+Camunda recommends storing shared templates in a separate repository:
+
+- **Desktop Modeler**: Copy templates manually into your global directory.
+- **Web Modeler**: Use a [CI/CD pipeline](/components/best-practices/cicd-guidelines/element-templates-at-scale.md) to sync templates with your repository via the [Web Modeler API](/apis-tools/web-modeler-api/index.md).
+
+### Project element templates
+
+| Desktop Modeler                                                                                                                                                                                | Web Modeler                                                                                                                                  |
+| ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------- |
+| [Local element templates](/components/modeler/desktop-modeler/element-templates/configuring-templates.md#global-templates) are loaded from the `.camunda/element-templates` folder if present. | Processes templates from a single folder. <br></br>If working with a project (not a process application), templates must first be published. |
+
+:::note
+
+- If starting in **Desktop Modeler**, use a single folder for your process application. This makes project templates available in both modelers without extra work.
+- If starting in **Web Modeler**, manually create an empty JSON object `{}` in a file named `.process-application` after cloning the repository so Desktop Modeler can correctly recognize the project.
```

Comment:
❓ I can do that in Web Modeler? Probably you mean, I need to create that outside of web modeler, right?
<img width="956" height="469" alt="image" src="https://github.com/user-attachments/assets/c382bc22-ab48-41a5-8144-d452bd0a0360" />